### ISSUE TITLE
Applications Menu don't open correctly categories view

### ISSUE THREAD

**Issue by btd1337** [elementary/applications-menu#106]
In the first opening after boot the applications menu don't open the category view.

This error occurs:

```
Jun 14 10:53:47 io.elementary.t[3202]: gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkNotebook
Jun 14 10:53:47 io.elementary.t[3202]: gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkNotebook
Jun 14 10:53:47 io.elementary.t[3202]: gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkNotebook
Jun 14 10:55:02 io.elementary.cerbere.desktop[1680]: VM 0x7f9b41700000 on pid 2280 received NeedDebuggerBreak trap
```
After the first opening and selecting the category view, the error does not occur again.

**Comment by jacob1237** [comment]
My setup is also affected.

Here is how it looks like: 
![2018-10-25 21-34-50](https://user-images.githubusercontent.com/1957938/47523102-ef3dbb00-d89f-11e8-8ea6-54205ef48d29.png)

This happpens only once after launch of wingpanel. To reproduce it just execute `killall wingpanel` and wait until wingpanel restart.

**Comment by peteruithoven** [comment]
Looks like this is a duplicate of: #146 
This has been fixed with: https://github.com/elementary/applications-menu/pull/149
There has been no new release yet of applications menu, so you'll still experience the issue for now
